distribution/docs/src/main/resources/_contents/_securing/managing-certificates-contents.adoc

```diff
@@ -0,0 +1,357 @@
+
+==== Certificate Management
+
+${branding} uses certificates in two ways:
+
+["lowerroman"]
+. Transmitting and receiving encrypted messages.
+. Authenticating an incoming user request.
+
+For more certificate configuration options, see <<_configuring_${ddf-security}, Configuring ${ddf-security}>>.
+
+===== Keystores
```

Comment:
I think a better title is "Keystore Management". I need to refer to the title in another section, so the more descriptive/unique the title, the better.